Gruntfile.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sort-keys */
```

Comment:
Because they were too many and I wanted to do it once this PR is merged to avoid future conflicts.. Maybe I'll add a TODO to do it in a separate PR after we merge this .